5.6/Dockerfile

```diff
@@ -20,14 +28,18 @@ is probably as a replacement for CGI scripts."
 LABEL summary="$SUMMARY" \
       description="$DESCRIPTION" \
       io.k8s.description="$DESCRIPTION" \
-      io.k8s.display-name="Apache 2.4 with PHP 5.6" \
+      io.k8s.display-name="Apache 2.4 with PHP $PHP_VERSION" \
       io.openshift.expose-services="8080:http" \
-      io.openshift.tags="builder,php,php56,rh-php56" \
-      name="centos/php-56-centos7" \
-      com.redhat.component="rh-php56-docker" \
-      version="5.6" \
+      io.openshift.tags="builder,$NAME,$NAME$PHP_VER_SHORT,rh-$NAME$PHP_VER_SHORT" \
+      io.openshift.s2i.scripts-url="image:///usr/libexec/s2i" \
+      io.s2i.scripts-url="image:///usr/libexec/s2i" \
+      name="centos/$NAME-$PHP_VER_SHORT-centos5" \
+      com.redhat.component="rh-$NAME$PHP_VER_SHORT-docker" \
+      version="$VERSION" \
       release="1" \
-      maintainer="SoftwareCollections.org <sclorg@redhat.com>"
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>" \
+      help="For more information visit https://github.com/sclorg/s2i-$NAME-container" \
+      usage="s2i build <SOURCE-REPOSITORY> $NAME:5 <APP-NAME>"
```

Comment:
No image with name `$NAME:5`. `centos/$NAME-$PHP_VER_SHORT-centos7:latest` instead?
(same for .rhel7 Dockerfile)